Source/BSN.Commons/Entities/Gender.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Commons.Users
+﻿namespace Commons.Entites
```

Comment:
Why name of directory is changed? I think Users namespace and directory is better than Entities.

Entities is more general and it is bad.